docs/user-guide/map-templates.md

```diff
@@ -0,0 +1,66 @@
+# Map Templates
+*******************
+
+The **Map Templates** is an extension added to the context to browse the map template like the *WMC templates* and the *MapStore JSON*. The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates. 
+
+<img src="../img/map-templates/map-templates-panel.jpg" class="ms-docimage"/>
+
+For each template in the *Map Templates* list the following buttons are displayed:
+
+<img src="../img/map-templates/map-templates-buttons.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+* The **Replace** button <img src="../img/button/replace-button.jpg" class="ms-docbutton" /> allows the user to replace the current map context with the template, as follows:
+
+<img src="../img/map-templates/replace-map.gif" class="ms-docimage"/>
+
+* The **Add Template** button <img src="../img/button/add_to_map_button.jpg" class="ms-docbutton" /> allows the user to add the template to current map context, as follows:
+
+<img src="../img/map-templates/add-templates.gif" class="ms-docimage"/>
+
+* The **Add to favorites** button <img src="../img/button/favourites-button.jpg" class="ms-docbutton" /> allows the user to add the template to favorites on top of the list
+
+## Enabling the Map templates in a context
+
+The *Map templates* extension is enabled by the admin in the [Application Context](application-context.md) wizard. In particular, this happens in the [third step](application-context.md#configure-plugins) of the wizard and after the extension has been added to the *Enabled Plugins* list.
+
+<img src="../img/map-templates/configure-templates.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+As soon as the **Configure templates** button <img src="../img/button/configure-templates-button.jpg" class="ms-docbutton" /> is selected the *Configure templates* window opens allows the admin to manage the templates.
+
+<img src="../img/map-templates/map-templates-window.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+Here the admin can select the templates to moving from the *Available Templates* list to the *Enabled Templates* list through the central bar, as follows:
+
+<img src="../img/map-templates/moving-templates.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+### Uploading the template
+
+In order to upload a new template the admin can select the **Upload new template** button <img src="../img/button/upload-button.jpg" class="ms-docbutton"/> and the **Upload new template** window appears:
+
+<img src="../img/map-templates/upload-map-template.jpg" class="ms-docimage" style="max-width:300px;"/>
+
+Here the admin, in order to import the file, can drag and drop it inside the import screen or select it from the folders of the local machine selecting the import screen.
```

Comment:
Here the admin, in order to import a template file, can drag and drop it inside the import area or simply click on that area to select it from the folders of the local machine.